lms/djangoapps/courseware/tests/test_masquerade.py

```diff
@@ -281,6 +282,26 @@ def get_progress_detail(self):
         return json.loads(self.look_at_question(self.problem_display_name).content)['progress_detail']
 
     @patch.dict('django.conf.settings.FEATURES', {'DISABLE_START_DATES': False})
+    def test_masquerade_as_specific_user_on_self_paced(self):
+        """
+        Test masquerading as a specific user for course info page when self paced configuration
+        flag is ON in LMS ADMIN side .
```

Comment:
`when self paced configuration flag is ON in LMS ADMIN side .` => `when self paced configuration "enable_course_home_improvements" flag is set`